src/ducks/Signals/signalModal/SignalMasterModalForm.js

```diff
@@ -1,46 +1,82 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { connect } from 'react-redux'
-import { Modal, Button, Icon } from '@santiment-network/ui'
+import { Button, Icon } from '@santiment-network/ui'
+import Dialog from '@santiment-network/ui/Dialog'
```

Comment:
Use either default import (better, because of the tree shaking) or import multiple exports